decidim-participatory_processes/app/models/decidim/participatory_process_role_config/valuator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcessRoleConfig
+    class Valuator < Base
+      def component_is_whitelisted?(manifest)
+        manifest.to_sym == :proposals
```

Comment:
Did you think about having this constant to be a configurable array, with `[:proposals]` as the default value? Does that makes any sense to you?

Reply:
I'm not sure I understood your comment, can you check the latest commits? @leio10